server/package.json

```diff
@@ -4,5 +4,26 @@
   "main": "app.js",
   "repository": "https://github.com/chingu-voyages/v8-bears-team-15.git",
   "author": "jidemobell <jidd32@gmail.com>",
-  "license": "MIT"
+  "license": "MIT",
+  "devDependencies": {
+    "colors": "^1.3.3",
+    "dotenv": "^7.0.0",
+    "eslint": "^5.16.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-node": "^8.0.1",
+    "eslint-plugin-promise": "^4.1.1",
+    "eslint-plugin-standard": "^4.0.0",
```

Comment:
I see we have successfully add _eslint_ to the project. Should there also be an _eslint_ configuration file with specific rules?

Reply:
I included this https://www.npmjs.com/package/eslint-config-standard 

and the project includes an eslint file that points to standard.

we can review this to use something everyone will prefer

Reply:
I see the `.eslintrc `file now. Found it the `development` branch